addons/l10n_mx/data/account_report_diot.xml

```diff
@@ -76,21 +179,84 @@
                     </record>
-                    <record id="tax_report_mx_diot_importation_16_tag" model="account.report.expression">
-                        <field name="label">importation_16</field>
+                    <record id="tax_report_mx_diot_16_imp_int_nc_tag" model="account.report.expression">
+                        <field name="label">paid_16_imp_int_nc</field>
                         <field name="engine">tax_tags</field>
-                        <field name="formula">DIOT: 16% IMP</field>
+                        <field name="formula">DIOT: 16% IMP INT NO ACREDITABLE</field>
+                    </record>
+
+                    <record id="tax_report_mx_diot_refunds_8_n_tag" model="account.report.expression">
+                        <field name="label">refunds_8_n</field>
+                        <field name="engine">tax_tags</field>
+                        <field name="formula">DIOT: Refunds 8% N.</field>
+                    </record>
+                    <record id="tax_report_mx_diot_refunds_8_s_tag" model="account.report.expression">
+                        <field name="label">refunds_8_s</field>
+                        <field name="engine">tax_tags</field>
+                        <field name="formula">DIOT: Refunds 8% S.</field>
+                    </record>
+                    <record id="tax_report_mx_diot_refunds_16_tag" model="account.report.expression">
+                        <field name="label">refunds_16</field>
+                        <field name="engine">tax_tags</field>
+                        <field name="formula">DIOT: Refunds 16%</field>
                     </record>
+                    <record id="tax_report_mx_diot_refunds_16_imp_tag" model="account.report.expression">
+                        <field name="label">refunds_16_imp</field>
+                        <field name="engine">tax_tags</field>
+                        <field name="formula">DIOT: Refunds 16% IMP</field>
+                    </record>
+                    <record id="tax_report_mx_diot_refunds_16_imp_int_tag" model="account.report.expression">
+                        <field name="label">refunds_16_imp_int</field>
+                        <field name="engine">tax_tags</field>
+                        <field name="formula">DIOT: Refunds 16% IMP INT</field>
+                    </record>
+
+                    <record id="tax_report_mx_diot_paid_8_n_wnc_tag" model="account.report.expression">
+                        <field name="label">paid_8_n_wnc</field>
+                        <field name="engine">domain</field>
+                        <field name="formula" eval="['|', '|', '|', ('tax_tag_ids', '=', '+DIOT: 8% N.'), ('tax_tag_ids', '=', '-DIOT: 8% N.'), ('tax_tag_ids', '=', '+DIOT: 8% N. NO ACREDITABLE'), ('tax_tag_ids', '=', '-DIOT: 8% N. NO ACREDITABLE')]"/>
+                        <field name="subformula">sum</field>
+                    </record>
```

Comment:
Could we express this as an aggregation of several other expressions using `tax_tags` formulas? (even if you make them hidden/not used anywhere else)?
Using domain formulas is actually less efficient because the queries aren't batched, whereas all tax_tags formulas are batched into a single query that does a GROUP BY on tax tags.